decentraland/unity-explorer
# Fix: removed category filter when swapping from wearables to emotes

## What does this PR change?

Fix #1013 removed the category filter set in wearables when swapping to emotes

Additionally improved backpack stability by tackling various issues and crashes

...

## How to test the changes?



1. Launch the explorer
2. Go to backpack
3. Go to emotes
4. Check base emotes are visible
5. Go back to avatar
6. Set a filter for wearables
7. Go to emotes
8. Check base emotes are visible

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
